### PR TITLE
Allowed users

### PR DESCRIPTION
**Pull request by soapy1**
Hey stakater team, this project is really awesome!
I want to use it but also be able to specify a list of allowed users that can go through the proxy. I figured it couldn't be that hard of a feature to add so I gave it a go. Maybe this was a naive idea? I've also never programmed in Go before so sorry if I did some really ridiculous things.

### CODE REVIEW

**Comment by stakater-user** [comment]
@soapy1 Image is available for testing. ``docker pull stakater/gitwebhookproxy:SNAPSHOT-PR-65-2``

**Comment by stakater-user** [comment]
@soapy1 Image is available for testing. ``docker pull stakater/gitwebhookproxy:SNAPSHOT-PR-65-5``

**Comment by kahootali** [comment]
Hey @soapy1 , thanks for contributing. Can you please add this flag in Readme and in helm templates as well

**Comment by stakater-user** [comment]
@soapy1 Image is available for testing. ``docker pull stakater/gitwebhookproxy:SNAPSHOT-PR-65-6``

File: pkg/proxy/proxy.go [hunk 1]
```diff
@@ -61,7 +62,21 @@ func (p *Proxy) isIgnoredUser(committer string) bool {
...
+}
+
+func (p *Proxy) isAllowedUser(committer string) bool {
+	if len(p.allowedUsers) > 0 {
+		if exists, _ := utils.InArray(p.allowedUsers, committer); exists {
+			return true
+		}
+	}
+
+	if (committer == "" && p.provider == providers.GithubName){
```
Comment (kahootali) [inline pkg/proxy/proxy.go]: @soapy1 This check should be removed, as if committer is empty, we would not want it to be considered as an allowedUser

**Comment by kahootali** [comment]
@soapy1 Also please update the values.yaml.tmpl file in templates/chart folder, as that file actually generates values.yaml in pipeline

**Comment by soapy1** [comment]
@kahootali, pushed up changes, thanks for the review :)

**Pull request merged**